### PR TITLE
[doc] Update README.md

### PR DESCRIPTION
**Pull request by VitaliyLi**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
- [ ] Did you read the [contributor guideline](https://github.com/facebookresearch/fairscale/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

## What does this PR do?
Fixes # (issue).

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by Vittorio-Caggiano — CHANGES_REQUESTED** [review]
should we have FairScale with capital F and S?

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 --------------------------------------------------------------------------------
 
 ## Description
-fairscale is a PyTorch extension library for high performance and large scale training for optimizing training on one or across multiple machines/nodes. This library extend basic pytorch capabilities while adding new experimental ones.
+fairscale is a PyTorch extension library for high performance and large scale training on one or multiple machines/nodes. This library extends basic PyTorch capabilities while adding new experimental ones.
```
Comment (min-xu-ai) [inline README.md:9]: did we decide on some capitalization?

**Review by min-xu-ai — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (min-xu-ai) [reply README.md:9]: agreed with @Vittorio-Caggiano

**Comment by min-xu-ai** [comment]
LGTM. I will merge it.

**Pull request merged**